albumentations/augmentations/blur/transforms.py

```diff
@@ -41,7 +41,7 @@ def apply(self, img: np.ndarray, ksize: int = 3, **params) -> np.ndarray:
         return F.blur(img, ksize)
 
     def get_params(self) -> Dict[str, Any]:
-        return {"ksize": int(random.choice(np.arange(self.blur_limit[0], self.blur_limit[1] + 1, 2)))}
+        return {"ksize": int(random.choice(np.arange(self.blur_limit[0], self.blur_limit[1] + 1, 2).tolist()))}
```

Comment:
use `range` instead of `np.arange`

Reply:
Have changed to 'range'. It seems a problem with the combination of 'random.choice' and np.arange,so i just fix 'arange' on these cases.